egs/hkust/s5/local/hkust_data_prep.sh

```diff
@@ -14,6 +14,11 @@ hkust_text_dir=$2
 train_dir=data/local/train
 dev_dir=data/local/dev
 
+# transcripts normalization and segmentation
+# needs external tools
+python -c "import mmseg" 2>/dev/null || {
```

Comment:
I think invoking this as python2.7 would be better, since mmseg doesn't work on python3, IIRC.  Obviously we should make sure that the script that actually uses mmseg later on also invokes it that way.

Reply:
Done.